libs/protocol-client/src/lib/client/types.ts

```diff
@@ -0,0 +1,5 @@
+import { GetUserCustomQuery, ListContributionsQuery } from '../protocol-types';
+
+export type ContributionItem = ListContributionsQuery['result'][number];
+
+export type UserData = GetUserCustomQuery['result'];
```

Comment:
It might be good to adopt a standard naming convention. Like ListContributionsQueryResult instead of contributionItem.